app/models/authentication.rb

```diff
@@ -118,15 +119,6 @@ def self.build_credential_options
     end
   end
 
-  def self.class_from_request_data(data)
```

Comment:
I didn't look, are there no other callers?  Can you grep not only manageiq but all providers to make sure we update all callers?

I usually do this now that we have so many extracted repos..

```
$ bundle show rails
/Users/joerafaniello/.gem/ruby/2.4.1/gems/rails-5.0.4
```

Take that path and basically grep the grandparent directory so it includes not only gems but bundler git gems:

`grep -ir "class_from_request_data" /Users/joerafaniello/.gem/ruby/2.4.1`

Or something like that